tests/renderer/setuptab.test.js

```diff
@@ -153,33 +160,36 @@ describe('Arguments form interactions', () => {
       },
     };
 
-    uiSpec = {order: [Object.keys(spec.args)]}
+    uiSpec = { order: [Object.keys(spec.args)] };
     fetchValidation.mockResolvedValue(
       [[Object.keys(spec.args), validationMessage]]
     );
   });
 
   test('Browse button populates an input', async () => {
     spec.args.arg.type = 'csv';
-    const { findByText, findByLabelText } = renderSetupFromSpec(spec, uiSpec);
+    const {
+      findByRole, findByLabelText
+    } = renderSetupFromSpec(spec, uiSpec);
 
-    const input = await findByLabelText(RegExp(`${spec.args.arg.name}`));
+    const input = await findByLabelText(`${spec.args.arg.name}`);
     expect(input).toHaveAttribute('type', 'text');
-    expect(await findByText('Browse')).toBeInTheDocument();
+    expect(await findByRole('button', { name: /browse for/ }))
+      .toBeInTheDocument();
 
     // Browsing for a file
     const filepath = 'grilled_cheese.csv';
     let mockDialogData = { filePaths: [filepath] };
     ipcRenderer.invoke.mockResolvedValue(mockDialogData);
-    fireEvent.click(await findByText('Browse'));
+    fireEvent.click(await findByRole('button', { name: /browse for/ }));

```

Comment:
Is there a reason to keep `fireEvent` here and not user `userEvent`?

Reply:
Nope, no reason. FireEvent is still used in tons places in our tests. I only switched to userEvent for that one test that I was touching a bit more heavily anyway. 

If we think it's worth doing, I'd say we make the switch everywhere in a dedicated PR.